bokehjs/src/coffee/models/scales/log_scale.coffee

```diff
@@ -2,24 +2,10 @@ import {Scale} from "./scale"
 import * as p from "core/properties"
 
 export class LogScale extends Scale
-
-  connect_signals: () ->
-    super()
-    @connect(@properties.source_range.change,       () -> @_state = null)
-    @connect(@properties.target_range.change,       () -> @_state = null)
-    @connect(@source_range.properties.start.change, () -> @_state = null)
-    @connect(@source_range.properties.end.change,   () -> @_state = null)
-    @connect(@target_range.properties.start.change, () -> @_state = null)
-    @connect(@target_range.properties.end.change,   () -> @_state = null)
-
-  @getters {
-    state: () ->
-      if not @_state? then @_state = @_compute_state()
-      return @_state
-  }
+  type: "LogScale"
 
   compute: (x) ->
-    [factor, offset, inter_factor, inter_offset] = @state
+    [factor, offset, inter_factor, inter_offset] = @_compute_state()
```

Comment:
As far as I can tell, this doesn't meaningfully impact performance. The "slow" part of mapping is converting a large array from data space into screen space, not computing the mapper state once at the beginning.